compiler/test/suites/records.re

```diff
@@ -21,7 +21,10 @@ describe("records", ({test, testSkip}) => {
   );
   assertRun(
     "record_multiple",
-    "export record Rec {foo: Number, bar: String, baz: Bool}; print({foo: 4, bar: \"boo\", baz: true})",
+    {|
+      export record Rec {foo: Number, bar: String, baz: Bool}
+      print({foo: 4, bar: "boo", baz: true})
+    |},
     "{\n  foo: 4,\n  bar: \"boo\",\n  baz: true\n}\n",
```

Comment:
This should at least use the alternative string syntax so we don't need to escape the quotes, even if we decide to keep the `\n`

Reply:
Yep, agreed.